lib/matplotlib/collections.py

```diff
@@ -1122,13 +1139,18 @@ class BrokenBarHCollection(PolyCollection):
     @docstring.dedent_interpd
     def __init__(self, xranges, yrange, **kwargs):
         """
-        *xranges*
-            sequence of (*xmin*, *xwidth*)
+        Parameters
+        ----------
+        xranges : sequence of (float, float)
+            The horizontal position and extent for each bar given as sequence
```

Comment:
The sequence of (left-edge-position, width) of each bar.
(no need to introduce xmin, xwidth as variables)